package.json

```diff
@@ -4,12 +4,25 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build:stencil": "cd stencil-components && npm run build && cd ../",
-    "build:react-components": "cd react-components && npm run build && cd ../",
-    "build:angular-components": "cd angular-components && npm run build && cd ../",
-    "build:vue-components": "cd vue-components && npm run build && cd ../",
-    "build": "npm-run-all -s build:* copy-stencil-output",
+    "install:components": "npm ci --prefix stencil-components && npm ci --prefix angular-components && npm ci --prefix storybook && npm ci --prefix vue-components && npm ci --prefix react-components",
+
+    "build:components:stencil": "cd stencil-components && npm run build && cd ../",
+    "build:components:react-components": "cd react-components && npm run build && cd ../",
+    "build:components:angular-components": "cd angular-components && npm run build && cd ../",
+    "build:components:vue-components": "cd vue-components && npm run build && cd ../",
+
+    "build:app:angular": "npm ci --prefix angular-app && npm run build --prefix angular-app",
+    "build:app:ember": "npm ci --prefix ember-app && npm run build --prefix ember-app",
+    "build:app:react": "npm ci --prefix react-app && npm run build --prefix react-app",
+    "build:app:storybook": "npm ci --prefix storybook && npm run build --prefix storybook",
+    "build:app:vue": "npm ci --prefix vue-app && npm run build --prefix vue-app",
```

Comment:
Nit: I see an opportunity to make the script names a bit more consistent. 

Pattern 1: `build:<type>:<framework>` (e.g. `build:app:vue`, `build:components:stencil`)
Pattern 2: `build:<type>:<framework>-<type>` (e.g. `build:components:vue-components`) 

Might it make sense to settle on one or the other? My preference is the `build:<type>:<framework>` version.

This is not a merge blocker. If this is too big of a hassle, feel free to ignore. 🙂

Reply:
Hmm, good question. Let me take a look and see how much work this would be to adjust.

Reply:
Done! https://github.com/cloudfour/tpp-research-monorepo/pull/14/commits/58d6261b4b26833c5960a6491407c9ab57ffd5e2